src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingSpec.cs

```diff
@@ -156,14 +191,15 @@ private Stop()
         public static readonly ShardResolver ExtractShardId = message =>
         {
             if (message is EntityEnvelope)
-                return (((EntityEnvelope)message).Id % ShardsCount).ToString();
+                return (((EntityEnvelope)message).Id % NumberOfShards).ToString();
```

Comment:
I got you point. Yes, we can make some code clean-up in the subsequent pull requests